server/rpc.go

```diff
@@ -177,7 +178,7 @@ func verifySegCreds(job *lpTypes.Job, segCreds string) (*SegData, error) {
 	return &segData, nil
 }
 
-func GetTranscoder(context context.Context, orch Orchestrator, req *TranscoderRequest) (*TranscoderInfo, error) {
+func GetTranscoder(context context.Context, orch Orchestrator, req *net.TranscoderRequest) (*net.TranscoderInfo, error) {
```

Comment:
Seems `GetTranscoder` is only used in `lphttp.GetTranscoder`.  Make it a private method?

Reply:
Fixed https://github.com/livepeer/go-livepeer/pull/512/commits/276c2c2ee4f2e58760cd458bf1670ab1cd32eb4a